### PR TITLE
Update other dependencies in mobile app

### PR DESCRIPTION
**Pull request by yksolanki9**

### CODE REVIEW

File: src/app/fyle/my-view-advance-request/my-view-advance-request.page.scss [hunk 1]
```diff
@@ -155,12 +155,6 @@
     padding: 8px 0px;
   }
 
-  &--row-items {
```
Comment (Julias0) [inline src/app/fyle/my-view-advance-request/my-view-advance-request.page.scss:158]: why are these changes here? How are these related to dependencies?

**Review by Julias0 — CHANGES_REQUESTED** [review]
minor questions

File: src/app/fyle/my-view-advance-request/my-view-advance-request.page.scss (hunk 1 above)
Comment (yksolanki9) [reply src/app/fyle/my-view-advance-request/my-view-advance-request.page.scss:158]: Saw these warnings when building the app, so fixed them

Comment (yksolanki9) [reply src/app/fyle/my-view-advance-request/my-view-advance-request.page.scss]: Created a separate pr for them

**Review by Julias0 — APPROVED** [review]
